src/Microsoft.OpenApi/Properties/SRResource.resx

```diff
@@ -210,6 +210,12 @@
   <data name="Validation_RuleAddTwice" xml:space="preserve">
     <value>The same rule cannot be in the same rule set twice.</value>
   </data>
+  <data name="Validation_SchemaDoesntContainDiscriminatorProperty" xml:space="preserve">
+    <value>Schema {0} doesn't contain discriminator property {1}.</value>
+  </data>
+  <data name="Validation_SchemaDoesntContainDiscriminatorPropertyInRequiredFieldList" xml:space="preserve">
```

Comment:
>Validation_SchemaDoesntContainDiscriminatorPropertyInRequiredFieldList [](start = 14, length = 70)

Suggest renaming this to:

Validation_CompositeSchemaRequiedFieldMustContainThePropertySpecifiedInTheDiscriminator